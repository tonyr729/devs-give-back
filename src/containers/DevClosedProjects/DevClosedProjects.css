.under-construction {
  background-image: url("../../images/09.svg");
  background-repeat: no-repeat; }
  .under-construction a {
    color: white;
    font-size: 2rem;
    text-align: right;
    clear: right;
    display: block;
    margin: 19rem 18rem 0 0; }
    .under-construction a:hover {
      color: #6a5acd; }

.warning {
  font-size: 3rem;
  text-align: right; }

.blah {
  font-size: 2rem;
  margin: 0 4rem;
  float: right;
  width: 40rem; }

.home {
  font-size: 2rem;
  margin: 0 4rem;
  float: right;
  width: 40rem; }
