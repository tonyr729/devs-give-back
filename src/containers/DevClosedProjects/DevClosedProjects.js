import React from 'react';
import './DevClosedProjects.css';
import { NavLink } from 'react-router-dom';

export const DevClosedProjects = () => {
  return (
    <div className="under-construction">
      <h1 className="warning">UNDER CONSTRUCTION!</h1>
      <p className="blah">
        Yayyy thanks for clicking that link but... 
        this isn't done yet. Probably because I'm 
        sitting here making this message when I could be coding it... 
        but whatever!
      </p>
      <NavLink to='/'>
        Uhh... take me back to the start?
      </NavLink>
    </div>
  );
};