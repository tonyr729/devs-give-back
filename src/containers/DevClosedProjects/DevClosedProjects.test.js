import React from 'react';
import { shallow } from 'enzyme';
import { DevClosedProjects } from "./DevClosedProjects";


describe('DevClosedProjects', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<DevClosedProjects/>);

    expect(wrapper).toMatchSnapshot();
  });
});