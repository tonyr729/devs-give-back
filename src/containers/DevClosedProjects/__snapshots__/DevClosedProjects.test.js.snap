// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DevClosedProjects should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1
          className="warning"
>
          UNDER CONSTRUCTION!
</h1>,
        <p
          className="blah"
>
          Yayyy thanks for clicking that link but... this isn't done yet. Probably because I'm sitting here making this message when I could be coding it... but whatever!
</p>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/"
>
          Uhh... take me back to the start?
</NavLink>,
      ],
      "className": "under-construction",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "UNDER CONSTRUCTION!",
          "className": "warning",
        },
        "ref": null,
        "rendered": "UNDER CONSTRUCTION!",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Yayyy thanks for clicking that link but... this isn't done yet. Probably because I'm sitting here making this message when I could be coding it... but whatever!",
          "className": "blah",
        },
        "ref": null,
        "rendered": "Yayyy thanks for clicking that link but... this isn't done yet. Probably because I'm sitting here making this message when I could be coding it... but whatever!",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": "Uhh... take me back to the start?",
          "to": "/",
        },
        "ref": null,
        "rendered": "Uhh... take me back to the start?",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="warning"
>
            UNDER CONSTRUCTION!
</h1>,
          <p
            className="blah"
>
            Yayyy thanks for clicking that link but... this isn't done yet. Probably because I'm sitting here making this message when I could be coding it... but whatever!
</p>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/"
>
            Uhh... take me back to the start?
</NavLink>,
        ],
        "className": "under-construction",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "UNDER CONSTRUCTION!",
            "className": "warning",
          },
          "ref": null,
          "rendered": "UNDER CONSTRUCTION!",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Yayyy thanks for clicking that link but... this isn't done yet. Probably because I'm sitting here making this message when I could be coding it... but whatever!",
            "className": "blah",
          },
          "ref": null,
          "rendered": "Yayyy thanks for clicking that link but... this isn't done yet. Probably because I'm sitting here making this message when I could be coding it... but whatever!",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "Uhh... take me back to the start?",
            "to": "/",
          },
          "ref": null,
          "rendered": "Uhh... take me back to the start?",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
